index.html

```diff
@@ -4,66 +4,61 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title> My Portfolio</title>
+  <title>Portfolio</title>
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Playfair+Display" />
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Poppins" />
 </head>
 <body>
   <div id="nav-wrapper">
     <nav>
-
-      
       <ul>
         <li><a href="#" rel="access">Hello</a></li>
         <li><a href="#" rel="acess">Portfolio</a></li>
         <li><a href="#" rel="access">About</a></li>
         <li><a href="#" rel="access">Contact</a></li>
       </ul>
     </nav>
-    <div class="icon" id="icon">
-      <div class="icon-1"></div>
-      <div class="icon-2"></div>
-      <div class="icon-3"></div>
+    <div class="burger" id="burger">
+      <div class="burger-1"></div>
+      <div class="burger-2"></div>
+      <div class="burger-3"></div>
     </div>
-    
     <section id="greeting" class="greeting">
-
-
       <h1>Mohammed Hussain</h1>
       <p>
         Hello! I am a software developer! I can help you build a product, feature
-        or website. Take a look of my work. If you like what you see and have a
+        or website. Taka a look of my works. If you like what you see and have a
         project you need coded, don’t hesitate and contact me.
       </p>
       <a class="collab-link" href="#" rel="noopener noreferrer">
         <div class="collab-button">Start Collaboration</div>
```

Comment:
- [ ] Kindly add the appropriate padding around the **_` Start Collaboraton`_** element for the desktop view as specified in the Figma design you chose 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/186740109-e9176817-fdd4-4fca-b79f-b0d15a2b2492.png)  | ![image](https://user-images.githubusercontent.com/63473584/186740167-86f68890-2796-42fe-9245-fb27ca1f57bd.png) |